homepage.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    
+    <div class="container">
+        <div class="toolbar">
+            <div class="logo">
+                <img src="images/debtext.png" alt="LOGO">
+            </div>
+            <div class="space"></div>
+            <button id="menu" class="button">
+                <img src="images/Menu@1x.png" alt="Header">
+            </button>
+        </div>
+
+        <div class="headline">
+            <h1 id="primarytext">Hey There. I'm Debby</h1>
+            <h2 id="supportingtext">I'm a Software Developer</h2>
+            <p id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <div class="socialmedia">
+                <button class="button">
+                    <img src="images/GitHub.png" alt="Github">
+                </button>
+                <button class="button">
+                    <img src="images/Linkedin.png" alt="Linkdin">
+                </button>
+                <button class="button">
+                    <img src="images/Angelist.png" alt="Angelist">
+                </button>
+                <button class="button">
+                    <img src="images/Twitter.png" alt="Twitter">
+                </button>
+                <button class="button">
+                    <img src="images/Medium.png" alt="Medium">
+                </button>
+            </div>
+            <div class="illustration">
+                <img src="images/Illustration.png" alt="Illustration">
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] Kindly add the right background image for this section and stick to the design as much as possible including the fonts, sizes, and margins

|Your design|Figma design|
|-|-|
|![Screenshot_20220629_205204](https://user-images.githubusercontent.com/25227022/176534997-42be9d9d-41c4-4dd3-a45c-2831b5321522.png)|![Screenshot_20220629_210335](https://user-images.githubusercontent.com/25227022/176535025-73484a6c-e995-4698-ae0e-f6766347c9fa.png)|